src/ToggleGroup/ToggleGroup.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
+import { ToggleGroup } from '../ToggleGroup';
+
+export const items = [
+  {
+    label: 'Item A',
+    value: 'itema',
+  },
+  {
+    label: 'Item B',
+    value: 'itemb',
+  },
+  {
+    label: 'Item C',
+    value: 'itemc',
+  },
+];
+
+export const onChange = item => console.log(`The item value is '${item.value}' and the label is '${item.label}'`);
+
+<Meta title="ToggleGroup" component={ToggleGroup} />
+
+# ToggleGroup
+
+## Description
+
+ToggleGroup.
+
+## Preview
+
+<Preview>
+  <Story name="Default">
+    <ToggleGroup items={items} onChange={onChange} />
+  </Story>
+</Preview>
```

Comment:
Maybe add a story with just one option so we know the styling is all ok?

Reply:
It's a toggle though, there will never be one option.